lang/py/src/avro/io.py

```diff
@@ -341,7 +341,7 @@ def read_timestamp_millis_from_long(self):
     """
     timestamp_millis = self.read_long()
     timedelta = datetime.timedelta(microseconds=timestamp_millis * 1000)
-    unix_epoch_datetime = datetime.datetime(1970, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc) 
+    unix_epoch_datetime = datetime.datetime(1970, 1, 1, 0, 0, 0, 0, tzinfo=timezones.utc) 
```

Comment:
@jcnnghm updated the code to include our version of UTC tzinfo class and a TST(test) tzinfo class in lib/timezone.py